### PR TITLE
fix: only format printing of comments for docblocks

### LINKED ISSUE #268: Comments changes
Input:
```php
/*
<a href="#"
    class="btn btn-info ladda-button"
    data-ui-component="unified-modal-opener"
    data-style="zoom-in"
    data-title="Title modal"
    data-size="md"
    data-header="true"
    data-footer="false"
    data-render="contact-form"
    data-view="Title"
    >
    <span class="ladda-label">Modal</span>
</a>

Options: data-render - contact-form, render, template-part
*/
```

Output:
```php
/**
 * <a href="#"
 * class="btn btn-info ladda-button"
 * data-ui-component="unified-modal-opener"
 * data-style="zoom-in"
 * data-title="Title modal"
 * data-size="md"
 * data-header="true"
 * data-footer="false"
 * data-render="contact-form"
 * data-view="Title"
 * >
 * <span class="ladda-label">Modal</span>
 * </a>
 *
 * Options: data-render - contact-form, render, template-part
 */
```

Prettier don't change comment type in this case https://prettier.io/playground/#N4Igxg9gdgLgprEAuEB6AVAHSgHgIYAEAFgE5wBmAvJiAMQ3YFMFgA2eAzh9SAEYxQC-KAFoAllHIQC7ACay8I3gFcYMaA0HMFMRcrEjIAWwAO0BDB7KoY8mLiyRRiAtYiIJhHBKbmBHYocMACerHA8AF4QEEbiUL7aeLoiMGIwYTwAKmlhBM6uCUwBIhxiEeE0RrKF-kmKRHB4st48MCTKcDXFUhDwJDzkeKwcnSCMiclkUM39NJCweGAwIlIkRl11IgBu9gDuWTmj40wAfMcEOBwmeIJsnNw0cgoi7LxwrDQnALIuQzioVxuZ1wqDwwOwAHkTKloBwkLVJggZgQRCxoLolisIGsADQEKYzPHwUzseAia4kGDYdCobAgHEgDwwqAcZCgPAkEgQXYABQ5CFZKDwWwgYlk9JAnBgyEGwzgDN4JEWAGs4DAAMrXMASADmyDaHQZsggYBlQxGDIkI0pPKVOqMeDNcoZACsOAAPABCSrAqo1eCMcAAMhI4E6LYzVCZVAAmcPykAU63IPh4XihaASkwkCQwADqYpgRGQAA4AAwM7MQEZ5pUmFPZuDWrZhhlkACO+jItrw9sdSFlEZGRjE+vaCdKUB1YQAispemGB+aE7peAXZEXkDGGW08GJWLqAMIxB0pqDmCXKEaZNOCwdwAC+D6AA

### LINKED ISSUE #303: Non-standard docblock causes formatting confusion
This causes debug-check to complain, but doesn't actually break anything.  Notice the extra asterisk on the opening comment for the docblock - without that one asterisk everything works fine.

Input
```php
<?php
class test {
    /***
    * Configures the tasks tabs (sub_menu)
    */
    protected function get_sub_menu( $page = '' )
    {
    }
}
```
Output
```php
<?php
class test
{
    /***
        * Configures the tasks tabs (sub_menu)
        */
    protected function get_sub_menu($page = '')
    {

    }
}
```

Expected output:
```php
<?php
class test {
    /***
     * Configures the tasks tabs (sub_menu)
     */
    protected function get_sub_menu($page = '')
    {

    }
}
```

### PR DESCRIPTION
**Pull request by mgrip**
fixes #303 fixes #268

Pretty much re-using the same logic from the js printer https://github.com/prettier/prettier/blob/master/src/language-js/printer-estree.js#L5383

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -181,56 +181,25 @@ const printers = {
...
-            join(
-              hardline,
-              linesToPrint.map(line => {
-                return ` *${line.length > 0 ? ` ${line}` : ""}`;
-              })
-            ),
-            hardline,
-            " */"
-          ]);
+          // otherwise we can't be sure about indentationt, so just print as is
```
Comment (nicoder) [inline src/index.js]: s/indentationt/indentation ?

**Comment by evilebottnawi** [comment]
I will fix type, add tests https://github.com/prettier/plugin-php/issues/268 (looks it is also fix this) and rebase :+1:

**Comment by evilebottnawi** [comment]
@mgrip hm, can you don't use `fork`?

**Comment by evilebottnawi** [comment]
Also looks like we can solve https://github.com/prettier/plugin-php/issues/268

**Comment by czosel** [comment]
@evilebottnawi Using forks should be fine as long as "allow edits from maintainers" is checked.

**Comment by mgrip** [comment]
@evilebottnawi @czosel should be all set now! I added the extra test case and fixed spelling error.

**Pull request merged**